main.cpp

```diff
@@ -55,14 +56,25 @@ video: Chapter 2 - Part 3
  */
 
 //2)
-void variableDeclarations()
+void variableDeclarations()
+

```

Comment:
You didn't declare 3 variables of type `double` in this function.
you declared:
- float
- int
- bool
- char
- [ ] revise